<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastaDialogBase</class>
 <widget class="QDialog" name="CadastaDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadasta QGIS plugin</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>186</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Cadasta Project Download Wizard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="step_1_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Step 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="project_combo_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="get_available_projects_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Get Available Projects...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>491</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>This is the read only project description that changes when the combo box above is changed.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="step_2_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Step 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Project selection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
